src/modules/common/Config.ts

```diff
@@ -80,7 +80,7 @@ export class Config implements IConfig
         // Parse the arguments
         const parser = new ArgumentParser();
         parser.add_argument('-c', '--config', {
-            default: "config.yaml",
+            default: "docs/config.example.yaml",
```

Comment:
No no no! That's not a valid default.

Reply:
Well, do you have any other good ideas?

Reply:
Leave it as is.

Reply:
What's the motivation behind this PR ?

Reply:
It's not important.
It is because of the convenience of starting npm from the user's point of view.
This PR can be closed.